### ISSUE TITLE
Passing a Buffer body will pollute req.body when used along with `processNested`

### ISSUE THREAD

**Issue by jeanbmar** [richardgirges/express-fileupload#290]
Proxies tend to pass multipart body as Buffer instances to express. (e.g. [@vendia/serverless-express](https://github.com/vendia/serverless-express/blob/87aa26f5395de9dc22591a817a476a006268fa73/src/event-sources/utils.js#L18) and [serverless-http](https://github.com/dougmoscrop/serverless-http/blob/f72fdeaa0d25844257e01ff1078585a92752f53a/lib/provider/aws/create-request.js#L38)).

This implementation will cause a disaster when the provided body is a Buffer and imo could lead to possible security issues:
https://github.com/richardgirges/express-fileupload/blob/40d027fc978d9353da7abcc20faa010761074dd9/lib/processNested.js#L8-L9
Each byte of the multipart data buffer is becoming a key on the body:

```
body {
  '0': 45,
  '1': 45,
  '2': 45,
  '3': 45,
  '4': 45,
  '5': 45,
  '6': 87,
  '7': 101,
  '8': 98,
  '9': 75,
  '10': 105,
  '11': 116,
  [...]
  name: 'Tset2',
  slug: 'tset',
}
```

I suggest creating a body from scratch to avoid this instead of reusing the body provided by express, as Multer [does](https://github.com/expressjs/multer/blob/63f487421baec6f20d6a0a38561698f80daa2cb6/lib/make-middleware.js#L28).

**Comment by jmikrut** [comment]
Hey @richardgirges — looks like this has been merged in, but I'm not seeing a new release that includes it. Will you be publishing a new version? 

Thanks!

**Comment by richardgirges** [comment]
Hi @jmikrut - this has been released in [v1.2.1](https://github.com/richardgirges/express-fileupload/compare/v1.2.1...master#diff-9db95ee9b6338481d9bc69cf735c8f3f2e0102f09c8e54ff422acd84e9508795)+.

I apologize, as the release notes did not have #291 listed. I've updated the release notes to reflect this update:
https://github.com/richardgirges/express-fileupload/releases/tag/v1.2.1

If you're running on Node v12+, I would recommend upgrading to the latest version (v1.3.1) to take advantage of the latest security patches.
